pom.xml

```diff
@@ -13,7 +13,7 @@
 		<maven.compiler.target>17</maven.compiler.target>
 		<mockito.version>5.12.0</mockito.version>
 		<slf4j.version>2.0.13</slf4j.version>
-		<cqframework.version>3.18.0</cqframework.version>
+		<cqframework.version>3.20.0</cqframework.version>
```

Comment:
need a pom version bump for translator commons so we can maintain separate versions for qdm vs fhir